backend/test/fixtures/pbmc3k-genesets.csv

```diff
@@ -16,6 +16,6 @@ summary test,,F5,
 summary test,,PIGU,
 geneset_to_delete,,,
 geneset_to_edit,,,
-fill_this_geneset,,RER1,
+fill_this_geneset,,,
```

Comment:
Did this change break the python tests?

Reply:
Yes. @bkmartinjr updated the python spec in the previous e2e PR, but there was a mistake that we need to correct here — this geneset should be empty (so that it can be filled). Help appreciated, when I changed the python test in the previous PR, it did not behave as I anticipated. 

Python error: https://github.com/chanzuckerberg/cellxgene/pull/2327/checks?check_run_id=3139825571#step:7:1124

Line in question: https://github.com/chanzuckerberg/cellxgene/blob/colinmegill/undo-redo-e2e/backend/test/test_server/unit/common/test_api.py#L609

Reply:
You probably have to update the tests to reflect the changes you're making here.  There are assertions being made against this file in the python test suite, so be cautious when changing this file:

https://github.com/chanzuckerberg/cellxgene/blob/f150401ebff6bee345a3fb32ea19d62cff646464/backend/test/test_server/unit/common/test_api.py#L520-L584